packages/styles/stories/Components/dialog/default-dialog.example.html

```diff
@@ -1,37 +1,28 @@
 
 <section class="fd-dialog-docs-static fd-dialog fd-dialog--active">
-    <div class="fd-dialog__content" role="dialog" aria-modal="true" aria-labelledby="dialog-title-1">
+    <div class="fd-dialog__content fd-dialog__content--s" role="dialog" aria-modal="true" aria-labelledby="dialog-title-1">
         <span class="fd-dialog__resize-handle"></span>
-        <header class="fd-dialog__header fd-bar fd-bar--header-with-subheader">
+        <header class="fd-dialog__header fd-bar">
             <div class="fd-bar__left">
-                <div class="fd-bar__element">
-                    Dialog header
-                </div>
                 <div class="fd-bar__element">
                     <h2 class="fd-title fd-title--h5" id="dialog-title-1">
                         Dialog title
                     </h2>
                 </div>
             </div>
         </header>
-        <div class="fd-dialog__subheader fd-bar fd-bar--subheader">
-            <div class="fd-bar__left">
-                <div class="fd-bar__element">
-                    Dialog subheader
-                </div>
-            </div>
-        </div>
```

Comment:
I don't see why the subheader is removed. It's an optional case and users can opt for it. It's good to have it as an example so people know they can have a subheader.